testsuite/forge-cli/src/main.rs

```diff
@@ -444,14 +444,22 @@ impl Test for FundAccount {
 
 impl PublicUsageTest for FundAccount {
     fn run<'t>(&self, ctx: &mut PublicUsageContext<'t>) -> Result<()> {
-        let client = ctx.client();
+        let runtime = Runtime::new().unwrap();
+        runtime.block_on(self.async_run(ctx))
+    }
+}
+
+impl FundAccount {
+    async fn async_run(&self, ctx: &mut PublicUsageContext<'_>) -> Result<()> {
```

Comment:
Yeah i'll need to investigate just having an async run function.

Reply:
I think we can plan to replace runtime block in forge to use async fn